src/backer/backering.py

```diff
@@ -296,6 +299,25 @@ def on_post(self, req, rep):
 
         cr = httping.parseCesrHttpRequest(req=req)
         serder = serdering.SerderKERI(sad=cr.payload, kind=eventing.Serials.json)
+
+        backer_identifier = serder.ked["b"]
+        # Confirm registry backer
+        if TraitCodex.RegistryBackers.value not in serder.ked["c"]:
+            raise falcon.HTTPBadRequest(falcon.HTTP_400)
+
+        contains_valid_seal = False
+        for item in serder.ked["a"]:
+            if (
+                item["bi"]
+                and item["bi"] in backer_identifier
+                and item["d"] == REGISTRAR_SEAL_SAID
```

Comment:
This doesn't work. If I have 2 seals: `[{bi: <different>, d: REGISTRAR_SEAL_SAID}, bi: <correct>, d: DIFFERENT_SEAL_SAID}]`

This won't work. The `bi` has to be correct in the same seal as `d: REGISTRAR_SEAL_SAID`.

Reply:
updated
ref here: c7be26f5668e2e1c938f663723d3ffc3b4eb817a